rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/MultiChannelSchedulerDAO.java

```diff
@@ -0,0 +1,130 @@
+package com.lithium.dbi.rdbi.recipes.scheduler;
+
+import com.lithium.dbi.rdbi.BindArg;
+import com.lithium.dbi.rdbi.BindKey;
+import com.lithium.dbi.rdbi.Mapper;
+import com.lithium.dbi.rdbi.Query;
+
+import java.util.List;
+
+/**
+ *
+ * This is a pure round robin scheduler. If the next channel up doesn't have data it returns immediately
```

Comment:
I'm not sure I understand what you mean here.  If channels A & B have no scheduled jobs, but channel C has dozens of scheduled jobs, what happens when I call ```reserveMulti```?  Do I have to call multiple times, working my way through the empty queues before I get the scheduled jobs in channel C?

Reply:
no, you should only get C at that point

Reply:
actually this is a misleading comment from the original POC that Phu did - it no longer applies.